app/src/main/res/layout/activity_main.xml

```diff
@@ -4,4 +4,98 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".activityPage.MainActivity"/>
\ No newline at end of file
+    tools:context=".activityPage.SignUpPageActivity">
+
+
+    <ImageView
+        android:id="@+id/imageView2"
```

Comment:
I think this is something we might have to fix in general, but please use more descriptive names for id's of ui elements, like you did with email address for example.

Same for line 77 and 89

Reply:
I wanted to call it Services at first and I did not know what the activity was about. Thus, I moved it inn that folder. I can moved it somewhere else.

Reply:
@darkheros12 Yeah, I think that would only make sense once we actually make services, which would be done through Android ViewModels.